c10/cuda/CUDAAllocatorConfig.h

```diff
@@ -13,6 +13,12 @@
 
 namespace c10::cuda::CUDACachingAllocator {
 
+enum Expandable_Segments_Handle_Type : int {
```

Comment:
Please no C style enums in 2025
```suggestion
enum class Expandable_Segments_Handle_Type : int {
```

Reply:
fixed in [5558213](https://github.com/pytorch/pytorch/pull/156074/commits/555821382123be927d193951bd791144977298e3)